doc/api/process.markdown

```diff
@@ -137,13 +137,29 @@ Here is an example that logs every unhandled rejection to the console
         // application specific logging, throwing an error, or other logic here
     });
 
-For example, here is a rejection that will trigger the `'unhandledRejection'`
+For example, here is a rejection that will trigger the 'unhandledRejection'
 event:
 
     somePromise.then(function(res) {
       return reportToUser(JSON.pasre(res)); // note the typo
     }); // no `.catch` or `.then`
 
+Here is an example of a coding pattern that will also trigger
+'unhandledRejection':
+
+    function SomeResource() {
+      // Initially set the loaded status to a rejected promise
+      this.loaded = Promise.reject(new Error('Resource not yet loaded!'));
+    }
+
+    var resource = new SomeResource();
+    // no .catch or .then on resource.loaded for at least a turn
```

Comment:
Yeah – Promises execute out of the [microtask queue](https://github.com/iojs/io.js/blob/v1.x/src/async-wrap.cc#L162), if I'm not mistaken, which are related to the execution of [nextTick](https://github.com/iojs/io.js/blob/v1.x/src/async-wrap.cc#L172), which interleaves [microtask runs](https://github.com/iojs/io.js/blob/v1.x/src/node.js#L343) with [nextTick callbacks scheduled inside nextTicks](https://github.com/iojs/io.js/blob/v1.x/src/node.js#L333-L335)... so we get into a foggy area with regards to what a "turn" represents. The thrust of it is clear, but the specifics get hairy; it might be good to document what we mean when we use the general term in terms of the specific actions someplace.

In any case, I might have just talked myself into writing a PR for timers.markdown (with apologies to @trevnorris). :)